recipes/openssl/1.x.x/test_package/digest.c

```diff
@@ -7,7 +7,7 @@
 #include <openssl/crypto.h>
 #include <openssl/ssl.h>
 #if defined(WITH_ZLIB)
-#include <zlib.h>
```

Comment:
This is needed as in 2.0 the graph does not expose transitive header or libs which is why the feature test through openssl is required

Reply:
Hi @prince-chrismc,
The original test here did not actually validate that OpenSSL was built with zlib functionality. I revised the test to confirm that the zlib support was present in OpenSSL. This support does not require client access to the zlib header file. Let me know if you have any questions or concerns. Thanks